realis.go

```diff
@@ -68,69 +73,94 @@ type realisClient struct {
 	client         *aurora.AuroraSchedulerManagerClient
 	readonlyClient *aurora.ReadOnlySchedulerClient
 	adminClient    *aurora.AuroraAdminClient
+	logger         *log.Logger
 }
 
-type option func(*RealisConfig)
+type RealisConfig struct {
+	username, password          string
+	url                         string
+	timeoutms                   int
+	binTransport, jsonTransport bool
+	cluster                     *Cluster
+	backoff                     *Backoff
+	transport                   thrift.TTransport
+	protoFactory                thrift.TProtocolFactory
+	logger                      *log.Logger
```

Comment:
This should not be part of config. rather hide logger component all together and provide a flag to turn on and off debug logs

Reply:
Addressed this in the last update. Now we internally attach a non printing logger as the default, but switch to one that prints to stdout when using the build tag `gocql_debug` much like gocql does. Performance can be an issue since I used ioutils.discard instead of making a NOOP logger, but gonna hold off on early optimizations 💯

Reply:
Had to change this in order to be able to give the user of the library control over how they want to implement the logger. Unfortunately, the way we use the config object means that we need to included a logger type in the config struct. We should revisit this decision down the line.